config/settings.js

```diff
@@ -51,11 +67,30 @@ window.spinnakerSettings = {
         region: cfDefaultRegion
       },
     }
+    titan: {
+      defaults: {
+        account: titanPrimaryAccount,
+        region: titanDefaultRegion
+      },
+    },
+    kubernetes: {
+      defaults: {
+        account: kubernetesPrimaryAccount,
+        namespace: kubernetesDefaultNamespace
+      },
+    }
+  },
+  whatsNew: {
```

Comment:
We can probably `whatsNew` from both here and from deck's settings.js - it's Netflix-specific.